certification/pyxis/submit_test.go

```diff
@@ -19,7 +19,9 @@ var _ = Describe("Pyxis Submit", func() {
 	mux.Handle("/api/v1/projects/certification/id/my-awesome-project-id/test-results", &pyxisTestResultsHandler{})
 	mux.Handle("/api/v1/projects/certification/id/my-image-project-id/images", &pyxisImageHandler{})
 	mux.Handle("/api/v1/projects/certification/id/", &pyxisProjectHandler{})
+	mux.Handle("/api/v1/images/id/updateImage", &pyxisImageHandler{})
 	mux.Handle("/api/v1/images/id/blah/", &pyxisRPMManifestHandler{})
+	mux.Handle("/api/v1/images/id/updateImage/", &pyxisImageHandler{})
```

Comment:
Is this a duplicate ?

Reply:
Not a duplicate, this is to force the new tests back to the `pyxisImageHandler` where the `PATCH` mock is at. Since, RPM manifest route is subset of the image route, I had to add this in. The below picture from swagger might make this make more sense.

![image](https://user-images.githubusercontent.com/88156657/191848666-0228fbc2-4f6e-48c2-960a-a067a2cd873d.png)